docs/app/pages/changelog/changelog.component.html

```diff
@@ -3,9 +3,47 @@
 <div class="container">
     <div class="row">
         <div class="col-md-10 col-md-offset-1 col-sm-12 col-sm-offset-0 m-t-lg">
-            
+
+            <ux-ebox>
+
+                <ux-ebox-header>
+                    <h3 class="changelog-title">UX Aspects (AngularJS) end of life</h3>
+                    <p class="changelog-date">December 31st 2021</p>
+                </ux-ebox-header>
+
+                <ux-ebox-content>
+                    <p>
+                        As AngularJS 1.x reaches end of life at the end of this year, we have now retired the AngularJS version
```

Comment:
```suggestion
                        Because [AngularJS 1.x has reached the end of its support window](https://blog.angular.io/discontinued-long-term-support-for-angularjs-cc066b82e65a), we have now retired the AngularJS version
```